README.md

```diff
@@ -268,4 +268,4 @@ https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner-direc
 https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md
 
 [warehouse#12965]: https://github.com/pypi/warehouse/issues/12965
-[OpenID Connect Publishing]: https://pypi.org/help/#openid-connect
```

Comment:
Will the PyPI-hosted section be going away? Would it be useful to still link to it from somewhere?

Reply:
I don't think it'll be going away, but it's probably going to change fragments (since the current fragment references OIDC). Given that, I figured I'd just drop it and then re-add it later once it's ready.

Reply:
Okay, marking this thread as resolved.